static/js/common/upload-addon.js

```diff
@@ -337,11 +337,14 @@
                       // Specific messages for unlisted addons.
                       var isSideload = $('#id_is_sideload').is(':checked') || $new_form.data('addon-is-sideload');
                       var automaticValidation = $('#create-addon').data('automatic-validation') || $new_form.data('automatic-validation');
+                      var passedValidation = (!v.signing_summary || (!v.signing_summary.low &&
+                                                                     !v.signing_summary.medium &&
+                                                                     !v.signing_summary.high));
                       if (!isListed) {
                         if (isSideload) {
                           $("<p>").text(gettext("Your submission will go through a manual review.")).appendTo(upload_results);
                         } else {
-                          if (warnings === 0) {
+                          if (passedValidation) {
```

Comment:
Not familiar enough with the process to be sure I understand what's going on. You only want to test signing validation summary here ? Will that always be present ?